### DISCUSSION TITLE
File Explorer

### DISCUSSION THREAD

**Discussion by WatchTheory** [Flow-Launcher/Flow.Launcher#619] [Q&A] [answered]
just wanted to know if  we can use a different file explorer other than the window explorer for flow. Launcher?

**Comment by jjw24** [comment, marked as answer]
You can do this with Everything plugin `pm install everything`, but Explorer plugin doesn't yet have this

**Reply by taooceros** [reply]
Do we want to add a setting for that as a global setting, and pass that to plugin via api?

**Reply by WatchTheory** [reply]
Yes, please. That would help alot.
